TravelerApp/TravelerAppCore/Controller/GetAverage.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TravelerAppCore.Models.Hotels;
+
+namespace TravelerAppCore.Controller
+{
+    public class GetAverage
```

Comment:
liczenie średniej powinno być w klasie root swoją drogą warto też zastanowić się nad nazwą root czy faktycznie opisuje czym jest ta klasa

Reply:
Zmieniono "Root" na "Hotel". Liczenie średniej znajduje się teraz we właściwości "AverageRates" z wykorzystaniem metody dostępu get;